server/commands/write.go

```diff
@@ -60,9 +60,14 @@ func (c *WriteCommand) validateTuplesets(ctx context.Context, req *openfgapb.Wri
 	}
 
 	for _, tk := range writes {
-		if _, err := tupleUtils.ValidateTuple(ctx, c.datastore, store, modelID, tk, dbCallsCounter); err != nil {
+		tupleUserset, err := tupleUtils.ValidateTuple(ctx, c.datastore, store, modelID, tk, dbCallsCounter)
+		if err != nil {
 			return serverErrors.HandleTupleValidateError(err)
 		}
+		err = validateHasDirectRelationship(tupleUserset, tk)
```

Comment:
This comment applies here and elsewhere in the scope of this PR.

Would you mind adding newlines between logically disjoint lines? Collapsing all whitespace between logical blocks makes the code harder to read. I know it's a nit pick, but we've been trying to do that in the project 👍 

I'd appreciate it!

Reply:
@jon-whit - absolutely, thank you for pointing that out 👍 I'll tidy that up this evening!

Reply:
Thank you!